src/root/views/GlobalThreeW/index.tsx

```diff
@@ -69,6 +72,9 @@ function GlobalThreeW(props: Props) {
     location,
   } = props;
 
+  const allRegions = useReduxState('allRegions');
+  const regionList = allRegions?.data?.results || [];
```

Comment:
Lets not create a new object on render function.
Why not just use optional chaining on regionList instead